src/nft/components/details/AssetDetails.tsx

```diff
@@ -318,10 +318,10 @@ export const AssetDetails = ({ asset, collection }: AssetDetailsProps) => {
   const formattedEthprice = isNftGraphqlEnabled
     ? formatEth(parseFloat(lastSalePrice ?? ''))
     : formatEthPrice(lastSalePrice) || 0
-  const formattedPrice = isNftGraphqlEnabled
-    ? formattedEthprice
-    : lastSalePrice
-    ? putCommas(parseFloat(formattedEthprice.toString())).toString()
+  const formattedPrice = lastSalePrice
+    ? isNftGraphqlEnabled
+      ? formattedEthprice
+      : putCommas(parseFloat(formattedEthprice.toString())).toString()
```

Comment:
I believe the correct order is still having the `isNftGraphqlEnabled` check first as the main branch for the if/else but then instead of checking for `formattedEthprice` check instead for `lastSalePrice`

Reply:
We actually can't do that simplification as it leads to an accidental truthy and nan being shown but the first example works as expected with no last sale price

Reply:
Honestly this whole block could be simplified. Let's replace this and `formattedEthprice` with the following which reduces a lot of redundancy
```
const formattedPrice = lastSalePrice
    ? isNftGraphqlEnabled
      ? formatEth(parseFloat(lastSalePrice ?? ''))
      : putCommas(formatEthPrice(lastSalePrice))
    : null